app/presenters/cancellation_presenter.rb

```diff
@@ -23,6 +23,7 @@ def cancellation_warnings
       t('users.delete.bullet_1', app: APP_NAME),
       t('users.delete.bullet_2_loa1'),
       t('users.delete.bullet_3', app: APP_NAME),
+      t('users.delete.bullet_4', app: APP_NAME),
```

Comment:
We're passing `app` ("login.gov"), but not using it in the localized string.

Should we...

- Use it in the localized string in place of the hard-coded "login.gov"?
- Not pass it here?

I'd lean toward the former if we tend to avoid hard-coding login.gov. The same could apply to the updated `delete_account.info` and `account_reset_granted.info` strings.

Reply:
+1 let's interpolate in with `%{app}`